build.gradle

```diff
@@ -223,7 +223,8 @@ test {
         }
     }
 
-    systemProperty "samjdk.use_async_io", "false"
+    systemProperty "samjdk.use_async_io_samtools", "true"
+    systemProperty "samjdk.use_async_io_tribble", "false"
     systemProperty "samjdk.intel_deflater_so_path", "build/libIntelDeflater.so"
     systemProperty "samjdk.compression_level", "1"
```

Comment:
Would be good if we could somehow enforce consistency between these `systemProperty` settings here and `applicationDefaultJvmArgs` above.